toolchain/sem_ir/ids.h

```diff
@@ -649,6 +649,7 @@ constexpr NameScopeId NameScopeId::Package = NameScopeId(0);
 
 // The ID of an instruction block.
 struct InstBlockId : public IdBase, public Printable<InstBlockId> {
+  // Constants for BlockValueStore<InstBlockId>.
```

Comment:
Maybe "Types" or "Typedefs" would be clearer than "Constants" that we're referring to `ElementType` and `ValueType` and not `Empty` etc.

Reply:
Done, with "Types" (hopefully there won't be non-type constants in the future :))